File: Day18/fibonacci_using_recursion.c
#include<stdio.h>

// Recursion method to calculate Fibonacci numbers 
int getfibonacci(int num){
    if(num <= 1){
        return num; //Base cases: Fibonacci(0) = 0,Fibonacci(1) = 1
    }
    return getfibonacci(num-1) + getfibonacci(num-2);
}
//Let's start
int main(){

    int num;
    printf("Enter the Number of term in Fibonacci series:  ");
    scanf("%d", &num);

    // Number of term in the Fibonacci series
    printf("Fibonacci series up to %d terms: ",num);
    for(int i = 0; i<num; i++){
        printf("%d ",getfibonacci(i));
    }
    return 0;
}

File: Day18/fibonacci_using_recursion.py
"""Recursion method to calculate Fibonacci numbers """
def get_fibonacci(num):

    """Base cases: Fibonacci(0) = 0,Fibonacci(1) = 1"""
    if num <= 1:
        return num

    return get_fibonacci(num-1) + get_fibonacci(num-2)


#Let's start
NUM= int(input("Enter the Number of term in Fibonacci series:  "))

# Number of term in the Fibonacci series
print("Fibonacci series up to ",NUM, "terms: ")

for i in range(0,NUM,1):
    print(get_fibonacci(i) , end=" ")
